data/baidu

```diff
@@ -6,8 +6,13 @@ baidustatic.com
 baifubao.com
 bdimg.com
 bdstatic.com
+bdydns.net
+dlnel.com
 duapps.com
+jomocdn.net
+jomocdn.net
```

Comment:
duplicate with the domain in `data/baidu#L12`

Reply:
Solved